### ISSUE TITLE
blank preview  video when i start record on iphone 

### ISSUE THREAD

**Issue by abdfawwaz** [DeltaCircuit/react-media-recorder#145]
hi  , i have a problem when start recording only , the video become blank and i cant see my self , but if i make a live stream without record its  work fine ! ,

**Comment by AshishDhama** [comment]
@abdfawwaz Please share more details like how you are implementing, I recently integrated with react and didn't face any issues.

**Comment by superbrewllc** [comment]
Yes, mine works fine on Safari too. May need more details about like  what browsers you are working, what device you havez!
